static/css/inbox.less

```diff
@@ -870,14 +870,63 @@ a.fa:hover {
         color: #000000;
       }
     }
+
+    .sub-actions {
+      background-color: #FFFFFF;
+      height: 56px;
+      padding: 0;
+      margin: 0 0 1px 0;
+      border-radius: 0;
+      border: none;
+      display: flex;
+      box-shadow: 0 -6px 12px rgba(0, 0, 0, 0.4);
+
+      .mm-icon-caption {
+        border: none;
+        line-height: 1;
+        color: @label-color;
+        max-width: none;
+        margin: 6px 0 0;
+        .reports-icon {
+          background-color: @unverified-color;
+          height: 24px;
+          width: 24px;
+          display: block;
+          margin: 0 auto 6px auto;
+
+          &.reports-icon-verify-error {
+            mask: url(../img/icon-status-times.svg) no-repeat 50% 50%;
+            -webkit-mask: url(../img/icon-status-times.svg) no-repeat 50% 50%;
+          }
+          &.reports-icon-verify-valid {
+            mask: url(../img/icon-status-check.svg) no-repeat 50% 50%;
+            -webkit-mask: url(../img/icon-status-check.svg) no-repeat 50% 50%;
+          }
```

Comment:
Masking seems like a complicated and potentially performance intensive solution. Would it work to inline the svg   and use css to change the colour of image from grey to green or red? This is the way the ResourceIcon service works and allows us to mark icons for deceased people grey:

```css
.deceased svg path {
  fill: @deceased-color;
}
```

Reply:
Definitely would work and one of the options I considered, but I thought it would raise objections.